src/frontend/src/components/NavBarV2/index.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from "react";
+import ENV from "src/common/constants/ENV";
+import Exclamation from "src/common/icons/IconExclamation.svg";
+import CloseIcon from "src/common/images/close-icon.svg";
+import HeaderLogo from "src/common/images/gen-epi-logo.svg";
+import { useUserInfo } from "src/common/queries/auth";
+import { ROUTES } from "src/common/routes";
+import UserMenu from "./components/UserMenu";
+import {
+  AnnouncementBanner,
+  AnnouncementText,
+  AnnouncementTextBold,
+  Bar,
+  ButtonLink,
+  HeaderContainer,
+  HeaderLogoContainer,
+  HeaderMaxWidthContainer,
+  HeaderTopContainer,
+  HeaderTopLinks,
+  MobileNavClose,
+  MobileNavCloseContainer,
+  MobileNavLink,
+  MobileNavLinkContainer,
+  MobileNavSeparator,
+  MobileNavToggle,
+  MobileNavTray,
+  OrgSplash,
+  TextLink,
+} from "./style";
+
+export default function NavBarV2(): JSX.Element {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  function toggleMobileNav() {
+    setMenuOpen(!menuOpen);
+  }
+
+  const { API_URL } = ENV;
+
+  const { data } = useUserInfo();
+
+  const group = data?.group;
+
+  const orgElements = <React.Fragment>{group?.name}</React.Fragment>;
+
+  function hasOrg(): JSX.Element | null {
+    if (group === undefined) {
+      return null;
+    } else {
+      return orgElements;
```

Comment:
Since `orgElements` can be succinctly defined in one line, might be better to inline it here rather than making a const.